tests/FakeItEasy.Tests/Core/FakeWrapperConfiguratorTests.cs

```diff
@@ -1,9 +1,12 @@
 namespace FakeItEasy.Tests.Core
 {
     using System.Linq;
+    using System.Reflection;
     using FakeItEasy.Core;
     using FakeItEasy.Creation;
+#if FEATURE_BINARY_SERIALIZATION

```

Comment:
Yes, I seem to have! Odd, since you'd think I would've just grabbed the line I removed from the top of the file.
Oh. Now that I think of it, I seem to remember typing the `#if FEATURE_` and relying on Intellisense. Or the way I use it, apparently Unintellisense. :angry: